src/payment/data/sagas.js

```diff
@@ -164,8 +165,7 @@ export function* handleFetchCaptureKey() {
     yield put(microformStatus(STATUS_LOADING)); // we are refreshing the capture key
     const result = yield call(PaymentApiService.getCaptureKey);
     yield put(captureKeyDataReceived(result)); // update redux store with capture key data
-    // TODO: Not needed for Stripe, likely refactor to have Stripe's own hadleFetchCaptureKey
-    // yield put(captureKeyStartTimeout());
+    yield put(captureKeyStartTimeout());
```

Comment:
I thought this was the PCI compliance for CyberSource security timeout for the credit card field, and we don't need this for Stripe